Documentation/Signoffs/detaildesgincamera.md

```diff
@@ -0,0 +1,137 @@
+# Detailed Design for Camera Subsystem within Primary Data Acquisition System
+
+## Big Picture
+
+The primary data acquisition system will utilize real-time camera sensors to collect images of sectors of parking spots in the lot behind Bell Hall, in which these images will then be transmitted via wireless communication to a server where the images will be processed by an artificial intelligence (AI) algorithm to determine the number of Red parking spots that are filled and unfilled by cars. This subsystem is a critical subsystem because these cameras will provide the primary data to the system that will determine the count of vehicles in a lot. The images from these cameras will be sent over to the server to be processed by an artificial intelligence (AI) algorithm to determine the number of available parking spots in the Bell Hall lot. This number will then be cross-checked with that derived from the secondary acquisition system to determine its accuracy.
+
+## Specifications
```

Comment:
There is no specification for the height of the camera mount. How do you know that you will be able to see the cars?